OpenRA.Game/Player.cs

```diff
@@ -138,7 +138,8 @@ public static string ResolvePlayerName(Session.Client client, IEnumerable<Sessio
 			{
 				var botInfo = botInfos.First(b => b.Type == client.Bot);
 				var botsOfSameType = clients.Where(c => c.Bot == client.Bot).ToArray();
-				return botsOfSameType.Length == 1 ? botInfo.Name : $"{botInfo.Name} {botsOfSameType.IndexOf(client) + 1}";
+				var botName = TranslationProvider.GetString(botInfo.Name);
+				return botsOfSameType.Length == 1 ? botName : $"{botName} {botsOfSameType.IndexOf(client) + 1}";
```

Comment:
The issue is translating here. The results of this function are used in networking code

Reply:
We should also be passing the numeric suffix as an argument to the translation instead of appending it manually.